package.json

```diff
@@ -43,6 +43,7 @@
     "tap": "^12.0.1"
   },
   "files": [
+    "clone.js",
```

Comment:
Looks like this was done in a1587eae7c5dcf74ccf6556ac7d6f704ea794804 (by adding it to the end).

I like it better this way: more alphabetical, fewer lines affected.